PocketKit/Sources/PocketKit/Collections/CollectionViewModel.swift

```diff
@@ -277,7 +253,7 @@ extension CollectionViewModel {
         // Check if item is a collection
         if let collection = story.item?.collection, featureFlags.isAssigned(flag: .nativeCollections) {
```

Comment:
similar to the other places, do we want to add a check for `collectionSlug` based on the `url`?

Reply:
yep that's been updated all over.